addons/stock/models/stock_move.py

```diff
@@ -195,9 +195,8 @@ def _compute_show_details_visible(self):
         """
         has_package = self.user_has_groups('stock.group_tracking_lot')
         multi_locations_enabled = self.user_has_groups('stock.group_stock_multi_locations')
-        consignment_enabled = self.user_has_groups('stock.group_tracking_owner')
 
-        show_details_visible = multi_locations_enabled or consignment_enabled or has_package
+        show_details_visible = multi_locations_enabled or has_package
```

Comment:
if you don't tick show detailed operations on the delivery picking and your stock move line are reserved on a quant with an owner, you don't see it